client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -35,6 +35,7 @@ import support from 'lib/url/support';
 import { registrar as registrarNames } from 'lib/domains/constants';
 import DesignatedAgentNotice from 'my-sites/upgrades/domain-management/components/designated-agent-notice';
 import Dialog from 'components/dialog';
+import { getCurrentUser } from 'state/current-user/selectors';
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Emails have been sent to {{strong}}%(oldEmail)s{{/strong}} and {{strong}}%(newEmail)s{{/strong}}. Please ensure they're both confirmed to finish this process.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).